datafusion/sqllogictest/test_files/window.slt

```diff
@@ -4879,3 +4879,21 @@ SELECT lead(column2, 1.1) OVER (order by column1) FROM t;
 
 query error DataFusion error: Execution error: Expected an integer value
 SELECT nth_value(column2, 1.1) OVER (order by column1) FROM t;
+
+statement ok
+drop table t;
+
+statement ok
+create table t(a int, b int) as values (1, 2)
+
+query II
+select a, row_number() over (order by b) as rn from t;
+----
+1 1
+
+# RowNumber expect 0 args.
+query error
```

Comment:
not an error on `main`

Reply:
Wild -- I double checked and you are right

```sql
DataFusion CLI v41.0.0
> create table t(a int, b int) as values (1, 2);
0 row(s) fetched.
Elapsed 0.015 seconds.

> select a, row_number() over (order by b) as rn from t;
+---+----+
| a | rn |
+---+----+
| 1 | 1  |
+---+----+
1 row(s) fetched.
Elapsed 0.006 seconds.

> select a, row_number(a) over (order by b) as rn from t;
+---+----+
| a | rn |
+---+----+
| 1 | 1  |
+---+----+
1 row(s) fetched.
Elapsed 0.002 seconds.
```